packages/features/bookings/Booker/components/hooks/useBookings.ts

```diff
@@ -93,7 +93,7 @@ export const useBookings = ({ event, hashedLink, bookingForm, metadata, teamMemb
     function refactorMeWithoutEffect() {
       const data = _instantBooking.data;
 
-      if (!data) return;
+      if (!data || !data.booking) return;
```

Comment:
data.booking is not present when host hasn't accepted the meeting

Reply:
I'm looking at the return of `getInstantBookingLocation` and the only thing it's returning is `booking`. Do we even need this line? Would there be any side effects if we just deleted it?

Reply:
yeah we need it to get the daily video call url from it and  set the InstantVideoMeetingUrl

Reply:
booking is set to ACCEPTED and returned only when host accepts it.